src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -7,6 +7,8 @@
     <TargetFrameworkNetCore>netcoreapp2.1</TargetFrameworkNetCore>
     <TargetFrameworkNet5Win>net5.0-windows10.0.17763.0</TargetFrameworkNet5Win>
     <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
+    <TargetFrameworkNet6>net6.0</TargetFrameworkNet6>
+    <TargetFrameworkNet6Win>net6.0-windows10.0.17763.0</TargetFrameworkNet6Win>
```

Comment:
there was a plan to remove `net5.0-windows10.0.17763.0`, do we want to add `net6.0-windows10.0.17763.0'  then remove both later? is that the plan moving forward?

Reply:
MAUI uses net6.0-windows10.0.19041.0. Should we move to that instead of net5.0

Reply:
> there was a plan to remove net5.0-windows10.0.17763.0, do we want to add net6.0-windows10.0.17763.0 then remove both later? is that the plan moving forward?

I think this would be decided in the conversion on how we want to GA new WAM.

> MAUI uses net6.0-windows10.0.19041.0. Should we move to that instead of net5.0

I don't quiet understand why? The client app needs to target net6.0-windows10.0.19041.0 to be able to use MAUI. But MSAL doesn't rely on any MAUI features, so it can just target net6.0. MSAL needs minimum -windows.177763 only so the old WAM will work. So a MAUI app targeting net6.0-windows10.0.19041.0 can use net6.0 MSAL with broker preview.

Reply:
OK. I suggest we do a mini samples test for iOS + Andorid + WinUI (MAUI) + UWP before merging this change into main. So that will reduce the risk of breaking.